presto-orc/src/main/java/com/facebook/presto/orc/stream/LongInputStreamV2.java

```diff
@@ -102,7 +102,8 @@ else if (EncodingType.PATCHED_BASE.ordinal() == enc) {
             readDeltaValues(firstByte);
         }
     }
-    int trapOffsetIdx = 100000;
+
+    int trapOffsetIdx = 100_000;
```

Comment:
Do you know why this is 100_000? Maybe define a static final for it?

Reply:
@yingsu00 I don't. This class like all other Aria readers requires substantial refactoring, but I'm not ready to do it yet.